manifests/install.pp

```diff
@@ -4,16 +4,31 @@
 #
 class consul::install {
 
+  case $::operatingsystem {
+    'windows': {
+      $binary_name = 'consul.exe'
+      $binary_mode = '0775'
```

Comment:
why does windows need different permissions here?

If we already know what the group should be, I'm ok with making 775 being the default for both oses. Does 555 not work for running windows binaries?